src/app/ui/alpha_slider.cpp

```diff
@@ -0,0 +1,128 @@
+// Aseprite
+// Copyright (C) 2024  Igara Studio S.A.
+//
+// This program is distributed under the terms of
+// the End-User License Agreement for Aseprite.
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "app/ui/alpha_slider.h"
+#include "ui/message.h"
+
+#include <algorithm>
+
+using namespace ui;
+using namespace app::gen;
+
+namespace app {
+
+void AlphaSlider::getSliderThemeInfo(int* min, int* max, int* value) const
+{
+  switch(getAlphaRange()) {
+    case AlphaRange::PERCENTAGE:
+      if (min) *min = 0;
+      if (max) *max = 100;
+      if (value) *value = std::round(((double)100)*((double)getValue())/((double)getMaxValue()));
+
+      return;
+    case AlphaRange::EIGHT_BIT:
+      return Slider::getSliderThemeInfo(min, max, value);
+  }
+}
+
+bool AlphaSlider::onProcessMessage(ui::Message* msg)
```

Comment:
Some messages handling are similar to the original `Slider::onProcessMessage`, wouldn't be possible to expand Slider (e.g. with some virtual methods that can be override in AlphaSlider) to customize the behavior and avoid duplicating code?